k8s/README.md

```diff
@@ -1,9 +1,27 @@
-## Build the pipeline
+# Business Create UI
 
+## How to build the openshift environment
 
-1. **Setup parameters** Alter, or copy the _bcrs-entities-create-ui-build-pipeline.param_ updating the values for your pipeline.
-2. **Create the pipeline** process the template referencing your parameter file.
+### Images
+This repo has implemented github actions to build and push the new image into our *tools* environment in OpenShift when a PR is merged. Significance in OpenShift environment:
+- no build config (image is built already)
+- no build/dev pipeline (image pushed into dev already)
 
-```bash 
-oc process -f templates/bcrs-entities-create-ui-pipeline.json -p bcrs-entities-create-ui-build-pipeline.param  | oc create -f -
-```
\ No newline at end of file
+You will still need to build the image stream (`oc create imagestream <name>`) and create tags for *dev, test, test-previous, prod and prod-previous* (`oc tag <namespace-tools>/<image name>:<source> <namespace-tools>/<image name>:<destination>`)
+
+### Deployment Objects
+There are 5 objects you need to create in OpenShift for the UI to run: deployment config, service, route, and config map. By updating the param file values in *bcrs-business-create-ui.dc-srvc-rt-cmap.param* to what you need for the OpenShift environment and running `oc process -f templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json --param-file=business-create-ui.dc-srvc-rt-cmap-kcmap.param  | oc create -f -` you will create all 5 objects.
```

Comment:
And secrets :)

Reply:
There are no secrets that need to be created for this repo so far I think?

Reply:
Also with one password I think we're moving away from storing anything in openshift secrets

Reply:
What is the 5th object then?

Reply:
Ah, there are 2 config maps.

Reply:
It's the keycloak config map - updated the sentence to include it :)